packages/documentation-site/patternfly-docs/content/layouts/compass/dashboard.tsx

```diff
@@ -0,0 +1,275 @@
+          <Tab eventKey={4} title={<TabTitleText>Configuration</TabTitleText>} />
+          <Tab eventKey={5} title={<TabTitleText>Test Page</TabTitleText>} />
+        </Tabs>
+      </CompassPanel>
+    </>
+  );
+
+  const westContent = (
+    <CompassPanel isPill>
+      <ActionList isIconList isVertical>
+        <ActionListGroup>
+          <ActionListItem>
+            <Tooltip content="Play">
+              <Button variant="plain" icon={<PlayIcon />} aria-label="Play" isCircle />
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Add">
+              <Button variant="plain" icon={<OutlinedPlusSquare />} aria-label="Add" isCircle />
+            </Tooltip>
+          </ActionListItem>
+        </ActionListGroup>
+        <ActionListItem>
+          <Tooltip content="Assistant">
+            <Button variant="plain" icon={<Icon style={{ translate: ".05em .1em; scale: 1.1" }}><RHAiExperienceIcon /></Icon>} aria-label="Assistant" isCircle className="pf-v6-m-ai-indicator" />
+          </Tooltip>
+        </ActionListItem>
+        <ActionListGroup>
+          <ActionListItem>
+            <Tooltip content="Help">
+              <Button variant="plain" icon={<OutlinedQuestionCircleIcon />} aria-label="Help" isCircle />
+            </Tooltip>
+          </ActionListItem>
+          <ActionListItem>
+            <Tooltip content="Copy">
+              <Button variant="plain" icon={<OutlinedCopy />} aria-label="Copy" isCircle />
+            </Tooltip>
+          </ActionListItem>
+        </ActionListGroup>
+      </ActionList>
+    </CompassPanel>
+  );
+
+  const userDropdownItems = (
+    <>
+      <DropdownItem key="group 2 profile">My profile</DropdownItem>
+      <DropdownItem key="group 2 user">User management</DropdownItem>
+      <DropdownItem key="group 2 logout">Logout</DropdownItem>
+    </>
+  );
+
+  const onDropdownSelect = () => {
+    setIsDropdownOpen(false);
+  };
+
+  const onDropdownToggle = () => {
+    setIsDropdownOpen(!isDropdownOpen);
+  };
+
+
+  const userDropdown = (
+    <Dropdown
+      isOpen={isDropdownOpen}
+      onSelect={() => onDropdownSelect}
+      onOpenChange={(isOpen: boolean) => setIsDropdownOpen(isOpen)}
+      popperProps={{ position: "right" }}
+      toggle={(toggleRef: React.Ref<MenuToggleElement>) => (
+        <MenuToggle
+          ref={toggleRef}
+          onClick={onDropdownToggle}
+          isExpanded={isDropdownOpen}
+          variant="plain"
+        >
+          <Flex
+            alignItems={{ default: "alignItemsCenter" }}
+            gap={{ default: "gapMd" }}
+          >
+            Aliyah Frazier
+            <Avatar src={imgAvatar} alt="" size="md" />
+          </Flex>
+        </MenuToggle>
+      )}
+    >
+      <DropdownList>{userDropdownItems}</DropdownList>
+    </Dropdown>
+  );
+
+  const northContent = <CompassHeader logo={<RHAutomationsLogo />} nav={navContent} profile={userDropdown} />;
```

Comment:
I left the same [comment in @kmcfaul's PR](https://github.com/patternfly/patternfly-org/pull/4848#discussion_r2501405799) but wdyt about adding a link around the logo?